src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1984,8 +1984,13 @@ void searchContextProviders
 * 0. Preparations
 * 1. Preconditions
 * 2. Get the complete list of entities from mongo
+*
+* RETURN VALUE
```

Comment:
I'm not a big fan about using bool, except for functions that are *truly* boolean, e.g.

    bool isBlack(int color) { return (color == 0)? true : false; }

However, if you really want it to be bool, no probs.